CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to contribute
+
+## **Did you find a bug?**
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/softwaresaved/fat/).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/softwaresaved/fat/new). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+## **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the pull request description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+## **Do you intend to add a new feature or change an existing one?**
+
+* Create a [new issue](https://github.com/softwaresaved/fat/new) suggesting your change.
+
```

Comment:
Include this as a point in this section: 
* When submitting an issue, please include a reproducible case that we can actually run.